torch/__init__.py

```diff
@@ -1772,12 +1772,14 @@ def registerOp(cls, op_key, full_schema, op_impl, dispatch_key):
 
 # TODO(ycao): Move ExportedProgram, Constraint, dynamic_dim etc to torch.compiler namespace
 
+from torch.compiler import Constraint
```

Comment:
maybe move this inside the export() function?

Reply:
It is part of torch.export()'s type annotation (see argument `Constraints`), so I can't move it inside export(). Are you worried about exposing this symbol at torch.* level? `__all__` does not have it.